src/packages/@workspaces/features/workspace-actions/layout/WorkspaceActions.svelte

```diff
@@ -0,0 +1,592 @@
+            />
+          {/if}
+        </button>
+      </div>
+      <!--  
+        New dropdown button for adding new api, collection and import Curl
+      -->
+      {#if userRole !== WorkspaceRole.WORKSPACE_VIEWER}
+        <Dropdown
+          zIndex={600}
+          buttonId="addButton"
+          bind:isMenuOpen={addButtonMenu}
+          options={addButtonData}
+        >
+          <Tooltip
+            title={"Add Options"}
+            placement={"bottom"}
+            distance={12}
+            show={!addButtonMenu}
+            zIndex={10}
+          >
+            <button
+              id="addButton"
+              class="border-0 p-1 border-radius-2 add-button"
+              on:click={() => {
+                addButtonMenu = !addButtonMenu;
+              }}
+            >
+              <img src={plusIcon} alt="" />
+            </button>
+          </Tooltip>
+        </Dropdown>
+      {/if}
+    </div>
+
+    <!-- LHS Side of Collection Enivironment & Test Flows -->
+    <div
+      class="d-flex flex-column collections-list"
+      style="overflow:hidden; margin-top:5px;  flex:1;"
+    >
+      <!-----Collection Section------>
+      <div
+        class="ps-1"
+        class:full-height={isExpandCollection && !isExpandEnvironment}
+        class:half-height={isExpandCollection && isExpandEnvironment}
+      >
+        <CollectionList
+          {scrollList}
+          bind:userRole
+          {onRefetchCollection}
+          {showImportCurlPopup}
+          {collectionList}
+          {isGuestUser}
+          {currentWorkspace}
+          {userRoleInWorkspace}
+          {activeTabPath}
+          {activeTabId}
+          {showImportCollectionPopup}
+          {onItemCreated}
+          {onItemDeleted}
+          {onItemRenamed}
+          {onItemOpened}
+          {onBranchSwitched}
+          {searchData}
+          {toggleExpandCollection}
+          bind:isExpandCollection
+        />
+      </div>
+
+      <hr class="mt-1 mb-0 ms-1 me-0" />
+
+      <!-- Environment Section -->
+
+      <div
+        class="ps-1"
+        class:full-height={isExpandEnvironment && !isExpandCollection}
+        class:half-height={isExpandCollection && isExpandEnvironment}
+      >
+        <EnvironmentList
+          loggedUserRoleInWorkspace={userRole}
+          onCreateEnvironment={_viewModel.onCreateEnvironment}
+          onOpenGlobalEnvironment={_viewModel.onOpenGlobalEnvironment}
+          onDeleteEnvironment={_viewModel.onDeleteEnvironment}
+          onUpdateEnvironment={_viewModel.onUpdateEnvironment}
+          onOpenEnvironment={_viewModel.onOpenEnvironment}
+          onSelectEnvironment={_viewModel.onSelectEnvironment}
+          currentWorkspace={activeWorkspace}
+          environments={$environments}
+          currentEnvironment={$activeEnvironment}
+          {searchData}
+          {toggleExpandEnvironment}
+          bind:isExpandEnvironment
```

Comment:
import this view model to `collectionpage.svelte` and use here using exports

Reply:
Resovled